### ISSUE TITLE
Removing MinDate MaxDate

### ISSUE THREAD

**Issue by agausachs** [Paul-DS/bootstrap-year-calendar#217]
If I set minDate or maxDate, how can I remove it after, like when calendar is initialized by default?

I tested, setMaxDate(null) but does nothing.

Thanks!

**Comment by Paul-DS** [comment]
Indeed, the `setMinDate` and `setMaxDate` methods expect a date object, so `null` is not managed.
I keep this issue open in order to fix in a future version.
For now you can just set a the max date using a date far in the future.
